reporter/src/main/kotlin/reporters/spdx/SpdxDocumentModelMapper.kt

```diff
@@ -66,12 +67,12 @@ object SpdxDocumentModelMapper {
         licenseTextProvider: LicenseTextProvider,
         params: SpdxDocumentParams
     ): SpdxDocument {
-        val spdxPackageIdGenerator = SpdxPackageIdGenerator()
         val packages = mutableListOf<SpdxPackage>()
         val relationships = mutableListOf<SpdxRelationship>()
 
+        val rootPackages = ortResult.getProjects(omitExcluded = true, includeSubProjects = false)
         val rootPackage = SpdxPackage(
-            spdxId = spdxPackageIdGenerator.nextId("root-package"),
+            spdxId = rootPackages.first().id.toSpdxId("Project"),
```

Comment:
It looks like only packages are added to the SPDX document, not projects. If this is correct it would be better to use a generic name here like just "root-package" to not give the impression that the packages are all dependencies of the randomly selected project. But then also previously the `nextId` call would not have been required.

Reply:
> It looks like only packages are added to the SPDX document, not projects.

That will actually change with https://github.com/oss-review-toolkit/ort/pull/5126 that I need to rebase onto this one after merge.